app/controllers/cwic_controller.rb

```diff
@@ -15,6 +15,12 @@ def download
     render content_type: 'text/html', layout: false
   end
 
+  def granule
```

Comment:
I will rename the route to ```/cwic/edsc_granule```. The path needs ```URL.decode``` so I guess it is better to put it in a separate action so that we are not accidentally 'decode' things that CWIC endpoints can't handle :)